suites/tipset/miner_flow.go

```diff
@@ -71,15 +71,15 @@ func TestMinerCreateProveCommitAndMissPoStChallengeWindow(t *testing.T, factory
 				minerActorID,
 				chain.Nonce(0), chain.Value(collateral),
 			),
-			types.MessageReceipt{ExitCode: exitcode_spec.Ok, ReturnValue: []byte{}, GasUsed: big_spec.Zero()},
+			types.MessageReceipt{ExitCode: exitcode_spec.Ok, ReturnValue: nil, GasUsed: big_spec.Zero()},
```

Comment:
Can someone confirm that this is correct? It looks like nil is what the specs actor returns, but there might be something I'm missing.

Reply:
Yes `nil` is the correct value. 

Note that lotus currently skips validation of the message receipt return field: https://github.com/filecoin-project/lotus/blob/testnet/3/chain/validation/factories.go#L44